span.go

```diff
@@ -80,7 +80,7 @@ func (s *Span) SetOperationName(operationName string) opentracing.Span {
 // SetTag implements SetTag() of opentracing.Span
 func (s *Span) SetTag(key string, value interface{}) opentracing.Span {
 	s.observer.OnSetTag(key, value)
-	if key == string(ext.SamplingPriority) && setSamplingPriority(s, value) {
+	if key == string(ext.SamplingPriority) && !setSamplingPriority(s, value) {
```

Comment:
If we decide to debug sample here, we set the "sampling.priority" tag so the agent knows that the root of the debug trace started here.